CONTRIBUTING.md

```diff
@@ -79,6 +79,7 @@ request description and tag it with **Caveat for Existing Projects**.
 This helps us to document our release notes.
 * You can run the command `make checklanguage` to ensure that documentation
 you've contributed contains language that is friendly and accessible.
+* For feature changes that would be included in the Release notes, be sure to modify the `CHANGELOG.md` file.
```

Comment:
```suggestion
* Be sure to modify the `CHANGELOG.md` file.
```

I don't know how potential contributors are supposed to know what qualifies as "features changes that would be included in the Release notes". There might be a better phrasing to make it more conditional, but I don't think the clause is necessary